Client/GymApplicationClient/GymApplicationClient/main.cpp

```diff
@@ -2,6 +2,6 @@
 using namespace std;
 
 int main() {
-	cout << "This is an initial file";
+	cout << "Hello World";
```

Comment:
Sucks I need you to makes strings a linked list and have the following methods in the class:
nvm I dont want you to do that.